modules/dnn/src/layers/max_unpooling_layer.cpp

```diff
@@ -27,9 +27,9 @@ class MaxUnpoolLayerImpl CV_FINAL : public MaxUnpoolLayer
     MaxUnpoolLayerImpl(const LayerParams& params)
     {
         setParamsFrom(params);
-        poolKernel = Size(params.get<int>("pool_k_w"), params.get<int>("pool_k_h"));
-        poolPad = Size(params.get<int>("pool_pad_w"), params.get<int>("pool_pad_h"));
-        poolStride = Size(params.get<int>("pool_stride_w"), params.get<int>("pool_stride_h"));
+        poolKernel = Size(params.get<int>("kernel_w"), params.get<int>("kernel_h"));
```

Comment:
Test https://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/22041 is failed:
```
C++ exception with description "OpenCV(3.4.7-pre) /build/precommit_linux64/3.4/opencv/modules/dnn/include/opencv2/dnn/dnn.inl.hpp:333: error: (-204:Requested object was not found) Required argument "kernel_h" not found into dictionary in function 'get'
" thrown in the test body.
```

Please check if we can always set kernel sizes at tf_importer. If not - we need some default value.